urdf_parser_plugin/package.xml

```diff
@@ -16,8 +16,13 @@
   <url type="repository">https://github.com/ros2/urdf</url>
   <url type="bugtracker">https://github.com/ros2/urdf/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_export_depend>ament_cmake_ros</buildtool_export_depend>
```

Comment:
We don't add a `buildtool_export_depend` ament_cmake_ros in other packages, even in this repository.  Honest question: is there a particular reason it is needed here?

Reply:
> Honest question: is there a particular reason it is needed here?

Honest answer: it's not really ever needed. I was *trying* to be pedantic here :)

A bunch of the ament_cmake export functions add `find_package(ament_cmake...)` to the generated `projectConfig.cmake`, meaning those packages need to be available to downstream packages that `find_package()` this one on the `buildtool` machine. For example, `urdf` should `<buildtool_export_depend>` `ament_cmake_core` and `ament_cmake_libraries`. 

```console
$ cd install/urdf/share/urdf/cmake
$ grep . -R -e "find_package"
./urdfConfig.cmake:# flag package as ament-based to distinguish it after being find_package()-ed
./ament_cmake_export_include_directories-extras.cmake:  find_package(ament_cmake_core QUIET REQUIRED)
./ament_cmake_export_dependencies-extras.cmake:find_package(ament_cmake_libraries QUIET REQUIRED)
./ament_cmake_export_dependencies-extras.cmake:# find_package() all dependencies
./ament_cmake_export_dependencies-extras.cmake:  find_package(ament_cmake_core QUIET REQUIRED)
./ament_cmake_export_dependencies-extras.cmake:      find_package("${_dep}" QUIET REQUIRED)
```

In practice it's not needed by any package because the debian `ros-distro-urdf` will install the buildtool depends, making them available to downstream packages. The only way I could see this being an issue is if someone built and installed `urdf` from source, then decided to uninstall it's `<build_depend>` and `<buildtool_depend>` dependencies because they're in theory no longer needed.

Funnily enough, `urdf_parser_plugin` is one of the few packages it's **not** needed because it doesn't use any `ament_cmake` functions or macros that add a `find_package(ament_cmake_...)` call to the `projectConfig.cmake` :upside_down_face:

Removed in 288413b

Reply:
Thanks for the detailed explanation!  Makes sense to me to remove it.